src/structures/SkyBlock/SkyblockInventoryItem.js

```diff
@@ -82,16 +75,40 @@ class SkyblockInventoryItem {
     this.rarity = parseRarity(this.loreArray[this.loreArray.length - 1]);
     /**
      * The amount of dungeon stars the item has (each star equates to a 10% stat boost while in dungeons)
-     * @author linearaccelerator
      * @type {number}
      */
-    this.dungeonStars = this.name.match(/(\u272a)/g) ? this.name.match(/(\u272a)/g).length : 0;
+    this.dungeonStars = data.tag.ExtraAttributes.upgrade_level ? data.tag.ExtraAttributes.upgrade_level : 0;
     /**
      * Dungeon gear score of the item (or null if not present)
      * @author linearaccelerator
      * @type {number}
      */
     this.gearScore = parseGearScore(this.loreArray) || null;
+    /**
+     * Amount of art of war books applied to the item
+     * @type {number}
+     */
+    this.artOfWar = data.tag.ExtraAttributes.art_of_war_count ? data.tag.ExtraAttributes.art_of_war_count : 0;
+    /**
+     * Rune
+     * @type {object}
+     */
+    this.rune = data.tag.ExtraAttributes.runes ? data.tag.ExtraAttributes.runes : null;
+    /**
+     * UUID of the item
+     * @type {string}
+     */
+    this.uuid = data.tag.ExtraAttributes.uuid ? data.tag.ExtraAttributes.uuid : '';
+    /**
+     * The amount of applied potato books
+     * @type {number}
+     */
+    this.hotPotatoBooks = data.tag.ExtraAttributes.hot_potato_count ? data.tag.ExtraAttributes.hot_potato_count : 0;
+    /**
+     * Is the item soulbound
+     * @type {number}
+     */
+    this.soulbound = data.tag.ExtraAttributes.donated_museum == 1 ? true : false;
```

Comment:
this can you just be `this.soulbound = data.tag.ExtraAttributes.donated_museum == 1`, it's gonna return true or false